client/src/pages/CreateOrganizationProfile.js

```diff
@@ -0,0 +1,325 @@
+  textAlign: "left",
+};
+
+const StyleInput = {
+  width: "100%",
+};
+
+const types = Object.values(filterOptions)[2].options;
+const { type, industry } = createOrganizationProfile;
+
+const initialState = {
+  state: {
+    modal: false,
+    options: [],
+    selected: "",
+  },
+  formData: {
+    title: "",
+    description: "",
+    tags: [],
+    type: type.default.value,
+    industry: industry.default.value,
+  },
+  errors: [],
+};
+
+const errorMsg = {
+  name: "Please include an organization name.",
+  email: "Please include an organization email.",
+  location: "Please include a location.",
+  type: "Please include an organization type",
+  industry: "Please include an organization industry",
+  interest: "Please include what your looking for",
+  privacy: "Please include our Privacy Policy",
+  conditions: "Please include Terms and Conditions",
+};
+
+const CreateOrgProfile = (props) => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [location, setLocation] = useState("");
+  const [global, setGlobal] = useState("");
+  const [volunteer, setVolunteer] = useState("");
+  const [staff, setStaff] = useState("");
+  const [investors, setInvestors] = useState("");
+  const [donations, setDonations] = useState("");
+  const [other, setOther] = useState("");
+  const [privacy, setPrivacy] = useState("");
+  const [conditions, setConditions] = useState("");
+  const [state, setState] = useState(initialState.state);
+  const [formData, setFormData] = useState(initialState.formData);
+  const [errors, setErrors] = useState(initialState.errors);
+  const { modal, selected, options } = state;
+
+  const handleInputChangeName = (e) => {
+    setName(e.target.value);
+  };
+  const handleInputChangeEmail = (e) => {
+    setEmail(e.target.value);
+  };
+  const handleInputChangeLocation = (e) => {
+    setLocation(e.target.value);
+  };
+  const handleInputChangeGlobal = (e) => {
+    setGlobal(e.target.value);
+  };
+  const handleInputChangeVolunteer = (e) => {
+    setVolunteer(e.target.value);
+  };
+  const handleInputChangeStaff = (e) => {
+    setStaff(e.target.value);
+  };
+  const handleInputChangeInvestors = (e) => {
+    setInvestors(e.target.value);
+  };
+  const handleInputChangeDonations = (e) => {
+    setDonations(e.target.value);
+  };
+  const handleInputChangeOther = (e) => {
+    setOther(e.target.value);
+  };
+  const handleInputChangePrivacy = (e) => {
+    setPrivacy(e.target.value);
+  };
+  const handleInputChangeConditions = (e) => {
+    setConditions(e.target.value);
+  };
+
+  const showModal = (setting) => (e) => {
+    setState({
+      ...state,
+      modal: !state.modal,
+      options: setting.options,
+      selected: setting.type,
+    });
+  };
+
+  const closeModal = (e) => {
+    setState({
+      ...state,
+      modal: !state.modal,
+      options: [],
+      selected: "",
+    });
+  };
+
+  const handleFormData = (field) => (e) => {
```

Comment:
I think we're using `react-hook-form`, @robinv85 can you confirm?

Reply:
This will be fixed by #399